notebooks/example.ipynb

```diff
@@ -200,6 +199,143 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the at using the `config.compare()` method to compare two two config files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0103f44",
+   "metadata": {},
+   "source": [
+    "The uwtools.api.config.compare() method has the following syntax:\n",
+    "\n",
+    "`uwtools.api.config.compare(config_1_path: Path | str, config_2_path: Path | str, config_1_format: str | None = None, config_2_format: str | None = None)→ bool`\n",
+    "\n",
+    "Recognized file extensions are: ini, nml, sh, yaml.\n",
+    "\n",
+    "Parameters:\n",
+    "\n",
+    "- config_1_path – Path to 1st config file\n",
+    "\n",
+    "- config_2_path – Path to 2nd config file\n",
+    "\n",
+    "- config_1_format – Format of 1st config file (optional if file’s extension is recognized)\n",
+    "\n",
+    "- config_2_format – Format of 2nd config file (optional if file’s extension is recognized)\n",
+    "\n",
+    "The method returns False if config files had differences, otherwise True."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b121dc29",
+   "metadata": {},
+   "source": [
+    "For example, lets compare the two nml files that contain different text."
```

Comment:
```suggestion
    "For example, let's compare two Fortran namelist files with differences:"
```
1. It might help readers here to connect the `uwtools` designator `nml` with "Fortran namelist", something they might not already be aware of.
2. It's a subtle distinction but, with structured configuration formats like Fortran namelist, YAML, etc., it's helpful to keep in mind that they are not treated just as _text_ -- the `compare` tool understands their structure and can compare information in a way that takes the structure into account -- and that also ignores insignificant differences like ordering or whitespace. For these reasons, I think the word "text" here might suggest something that isn't pertinent.